boot/zephyr/main.c

```diff
@@ -60,11 +60,12 @@ void main(void)
 	}
 
 	printk("Bootloader chain: 0x%x\n", rsp.br_image_addr);
-	vt = (struct vector_table *)(rsp.br_image_addr +
-				     rsp.br_hdr->ih_hdr_size);
+	vt = (struct vector_table *)(rsp.br_image_addr + TEXT_SECTION_OFFSET);
```

Comment:
This is the vector table of the app itself, right? That offset will depend not on the header, but on CONFIG_TEXT_SECTION_OFFSET, used by the Zephyr app.

Not sure how to best align both things here, which is why I had it defined per device.